src/components/PageLink/PageLink.tsx

```diff
@@ -0,0 +1,73 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  people: Person[],
+  person: Person,
+  isSelected: (person: Person) => boolean,
+};
+
+export const PageLink: React.FC<Props> = ({ people, person, isSelected }) => {
+  const mother = people.find((user) => user.name === person.motherName);
+  const father = people.find((user) => user.name === person.fatherName);
```

Comment:
You may consider to map the mother and father obj immediately after loading the data from API to avoid passing bulk data to child components. The Person Type is also giving a hint to same.